Automattic/wp-calypso
# Add Jetpack Starter to pricing page

Related to pbNhbs-5V7-p2

## Proposed Changes

* This PR adds Jetpack Starter to cloud.jetpack.com/pricing as one of the main bundles offered. It removes code that is unused and follow the design proposed in 0MMYGZ7bRaHuwMF3fFS2VC-fi-3402_25235.

NOTE: not everything in the file above is implemented here as it would require a minor refactor, and it also overlaps with another project

## Testing Instructions



* Checkout this branch locally
* Run `yarn start` and `yarn start-jetpack-cloud-p`
* Visit jetpack.cloud.localhost:3001/pricing?calypso_env=development and make sure you see the new product and the design matches the one in Figma
* Go through the purchase process and make sure it works as expected
<img width="1850" alt="image" src="https://user-images.githubusercontent.com/5714212/236342992-549432fd-c1d1-410e-ab31-e5683625c07f.png">
<img width="1850" alt="image" src="https://user-images.githubusercontent.com/5714212/236343025-08c22d93-1bd3-46da-b62f-e41031518efd.png">


## Pre-merge Checklist
